examples/batterystats/config.json

```diff
@@ -1,20 +1,20 @@
 {
   "type": "native",
   "devices": {
-    "nexus6p": {}
+    "j7duo2": {}
   },
   "replications": 3,
   "duration": 60000,
-  "monkeyrunner_path": "/home/user/Android/Sdk/tools/bin/monkeyrunner",
-  "systrace_path": "/home/user/Android/Sdk/platform-tools/systrace/systrace.py",
-  "powerprofile_path": "android-runner/example/batterystats/Scripts/power_profile.xml",
+  "monkeyrunner_path": "/home/thijmen/Library/Android/sdk/tools/bin/monkeyrunner",
+  "systrace_path": "python2 /Users/thijmen/Library/Android/sdk/platform-tools/systrace/systrace.py",
```

Comment:
The Systrace problem should be addressed in every config I think. I remember me having issues because Systrace was launched using python 3 instead of python 2. This issue can be resolved by prefixing the Systrace path with `python2`. Although this is an option, I think a solution inside of the code would be cleaner (less chance for users to miss configure).